.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,7 +3,7 @@ name: Bug report
 about: Create a report to help us improve
 title: '[Bug]'
 labels: bug
-assignees: 'bradtaniguchi, nhcarrigan'
+assignees:
```

Comment:
I think this property can be removed completely, rather than defined, but with no value. (same goes for the other 2 files)

Reply:
Had to confirm this (GH automatically adds the property even if empty)  - it *does* seem to work without the property tho :D

Reply:
Alright Brad! Should be good to go now! I removed the property as a whole on all three files!

Reply:
Awesome thanks!